src/libraries/System.ComponentModel.TypeConverter/src/System.ComponentModel.TypeConverter.csproj

```diff
@@ -119,7 +119,6 @@
     <Compile Include="System\ComponentModel\DataObjectMethodAttribute.cs" />
     <Compile Include="System\ComponentModel\DataObjectMethodType.cs" />
     <Compile Include="System\ComponentModel\DefaultBindingPropertyAttribute.cs" />
-    <Compile Include="System\ComponentModel\DesignerAttribute.cs" />
```

Comment:
I'm not seeing the delete for this file.

Reply:
Since the diff of the file is small enough and I used git mv to do it, it is shown as a rename on the diff:
https://github.com/dotnet/runtime/pull/40425/files#diff-3c1478e97792216f3370461a1209c913

Reply:
For some reason github is rendering as an add: https://github.com/dotnet/runtime/pull/40425/files#diff-cfe6da5dfffa7283c15c62c680231733

Also, the file still appears in your branch:  https://github.com/safern/runtime/blob/IComponentAttributes/src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/DesignerAttribute.cs

Reply:
Interesting... I used git mv... will update.

Reply:
Done. Good catch, thanks